lib/senders/templates/new_device_login.html

```diff
@@ -35,6 +35,29 @@ <h1 style="font-family: sans-serif; font-size: 21px; line-height: 29px; font-wei
     {{#if timestamp }}{{ timestamp }}<br/>{{/if}}
 </p>
 
+<!--Button Area-->
+<tr height="50">
+  <td align="center" valign="top">
+    <table border="0" cellpadding="0" cellspacing="0" height="100%" width="100%" id="email-button" style="-webkit-text-size-adjust: 100%; border-collapse: collapse; mso-table-lspace: 0pt; mso-table-rspace: 0pt; background-color: #0a84ff; border-radius: 4px; height: 50px; width: 310px !important;">
+      <tr style="page-break-before: always">
+        <td align="center" valign="middle" id="button-content" style="font-family: sans-serif; font-weight: normal; text-align: center; margin: 0; color: #ffffff; font-size: 20px; line-height: 100%;">
+          <!--[if mso]>
+          <v:roundrect xmlns:v="urn:schemas-microsoft-com:vml" xmlns:w="urn:schemas-microsoft-com:office:word" href="{{{link}}}" style="width:280px;height:40px;v-text-anchor:middle;" arcsize="10%" stroke="f" fillcolor="#0a84ff">
+            <w:anchorlock/>
+            <center>
+          <![endif]-->
+          <a href="{{{link}}}" id="button-link" style="font-family:sans-serif; color: #fff; display: block; padding: 15px; text-decoration: none; width: 280px; font-size: 18px; line-height: 26px;">{{t "Manage devices"}}</a>
```

Comment:
Please also add the button to the `.txt` version of the email here: https://github.com/mozilla/fxa-auth-server/blob/master/lib/senders/templates/new_device_login.txt

Existing Example: https://github.com/mozilla/fxa-auth-server/blob/f3261a613780cdec738069d262bc759c046041a3/lib/senders/templates/post_change_primary.txt#L5

Reply:
Also change the `link` variable in https://github.com/mozilla/fxa-auth-server/blob/a937c168f220996cb2094192663854f503f24c3b/lib/senders/email.js#L731 that should now be the same as:  https://github.com/mozilla/fxa-auth-server/blob/a937c168f220996cb2094192663854f503f24c3b/lib/senders/email.js#L836 (it should use `this.accountSettingsUrl`